### DISCUSSION TITLE
想问下，，改变了配置，如何在不关闭vim情况下，，生效？

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#889] [Q&A] [answered]
比方说，我修改了relativenumber = true,这个配置，如何在不关闭vim情况下，使其生效

**Comment by ayamir** [comment, marked as answer]
`:luafile %`

**Reply by Jint-lzxy** [reply]
Be aware that for `lazy.nvim` reloading is still not automatic. For plugins u need to explicitly do `:Lazy reload <ur-plugin>` after this.

**WARNING:** This feature is all very experimental, so use with caution. Things will likely break.

**Reply by wxzvim** [reply]
okok，，感谢感谢两位
